distribution/ddf-common/src/main/resources/security/default.policy

```diff
@@ -306,6 +306,22 @@ grant codeBase "file:/org.apache.cxf.cxf-rt-bindings-soap/org.apache.cxf.cxf-rt-
     permission java.security.SecurityPermission "insertProvider";
     permission java.io.FilePermission "${ddf.home.perm}etc${/}application-definitions", "read";
     permission java.io.FilePermission "${ddf.home.perm}etc${/}application-definitions${/}*", "read";
+    permission java.io.FilePermission "${ddf.home.perm}etc${/}org.apache.karaf.shell.cfg", "read,write";
+    permission java.io.FilePermission "${ddf.home.perm}etc${/}org.apache.karaf.management.cfg", "read,write";
```

Comment:
❓ How many of the bundles in this block need these permissions?

Reply:
I would guess that all or nearly all of them do. These bundles are the standard chain of bundles that get hit on your way through Jetty and CXF.